bs-css/src/Css_AtomicTypes.re

```diff
@@ -1018,6 +1032,17 @@ module DistributedAlignment = {
     | `stretch => "stretch";
 };
 
+module LegacyAlignment = {
+  type t = [ | `legacy | `legacyRight | `legacyLeft | `legacyCenter];
+
+  let toString =
+    fun
+    | `legacy => "legacy"
+    | `legacyRight => "legacy right"
+    | `legacyLeft => "legacy left"
+    | `legacyCenter => "legacy center";
+};
```

Comment:
Alternatively, we can do something like:
```reason
type align = [ | `auto | `right | `left | `center ]
type t = [ | `legacy(align) ]

...
```
But the problem is that there is actually no `legacy auto`. What do you think?

Reply:
what you did seems more accurate and easier to use from the user perspective